app/styles/component/_post-search-results.scss

```diff
@@ -40,6 +40,24 @@
     justify-content: center;
     margin: 18px;
   }
+
+  &__disclaimer {
+    color: $wds-fandom-color-dark-gray;
+    font-size: $wds-font-size-xxs;
+    font-style: italic;
+    letter-spacing: .03px;
```

Comment:
Does this amount even make a visual difference? I wonder if this kind of stuff is intentional or just a weird byproduct of importing into Zeplin.

Reply:
I did notice a change and wanted to keep things up to spec for the designers. But I know what you mean.

Reply:
Will double check with design just to make sure.

Reply:
Looks like it was a cross-app translation error between Figma and Zeplin. The correct letter-spacing should be `.25rem`.